.env

```diff
@@ -1,2 +1,2 @@
 SECRET=someverysecretpassword
```

Comment:
Kindly add your `.env` to the `.gitignore` file

Reply:
In case you need these keys on Heroku or Travis, your can easily set them up on those platforms.

Reply:
ok, noted.